locales/fr-FR/messages.po

```diff
@@ -76,6 +85,12 @@ msgstr ""
 msgid "Low bandwidth"
 msgstr "Bande passante faible"
 
+msgid "Low upload speed"
+msgstr "Bande Passante non conforme"
+
+msgid "Max upload speed"
+msgstr "Bande Passante Montante Max"
```

Comment:
I did not pay attention, but the AC also mention a dot at the end of the sentence:
`"Bande Passante Montante Max."`